frontend/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Holiday</title>
+</head>
+
+<body>
+    <div id="holidayResult"></div>
+    <script type="text/javascript">
+        const result = document.getElementById('holidayResult');
+        document.addEventListener("DOMContentLoaded", function (event) {
+          const baseUrl = 'http://127.0.0.1:8383/api/showCalendar';
```

Comment:
const baseUrl = 'http://127.0.0.1:8383' and put outside function so all function can reuse?

Reply:
okay, changed